lib/net/ssh/multi/session.rb

```diff
@@ -447,7 +450,7 @@ def io_select_wait(wait)
 
     def keepalive_interval
       servers = server_list.select { |s| s.options[:keepalive] }
-      servers.map { |s| s.options[:keepalive_interval] }.compact.min
+      servers.map { |s| s.options[:keepalive_interval] || DEFAULT_IO_SELECT_TIMEOUT }.compact.min
```

Comment:
You're right, we need to go through Net::SSH to pick up the config file options.

I agree on the name too - I'll submit a pull request to add it shortly. Thanks for the explanation!